fence-edit.el

```diff
@@ -158,7 +158,7 @@ beginning of the line is needed to handle indentation."
     (line-beginning-position)))
 
 (defun fence-edit--next-line-beginning-position-at-pos (pos)
-  "Return the position of the beginning of the line after the line at POS.
+  "Return the position of the beginning of the line after the line at POS if a newline character is found there.
```

Comment:
Please wrap at 80 columns as specified in Elisp package style guidelines. If you use Flycheck, you can install `flycheck-package` to get feedback on these types of boring linting errors in realtime.

Reply:
Sorry about that, I thought `lispy-fill` would properly handle it but looking at it further it seems like it's not meant to deal with doc strings.